app/controllers/placements/support/schools/placements_controller.rb

```diff
@@ -1,14 +1,23 @@
 class Placements::Support::Schools::PlacementsController < Placements::Support::ApplicationController
   before_action :set_school
+  before_action :set_placement, only: %i[show]
 
   def index
     @pagy, placements = pagy(@school.placements.includes(:subjects, :mentors).order("subjects.name"))
     @placements = placements.decorate
   end
 
+  def show
+    @placement = @school.placements.find(params.fetch(:id)).decorate
```

Comment:
Note: This is being resolved in https://github.com/DFE-Digital/itt-mentor-services/pull/509/files